src/melody/melody-extension-core/operators.js

```diff
@@ -353,7 +353,7 @@ export const TestInstanceOfExpression = createTest(
 //endregion
 
 //region Utilities
-function createTest(text, typeName) {
+export function createTest(text, typeName) {
```

Comment:
now that we actually export this function, it might be better to actually rename it into something more descriptive like `createTestExpression`. what do you think?

Reply:
@zackad Agreed. I renamed the function to `createTestExpression`.